tests/unit-tests/tests-give.php

```diff
@@ -184,6 +184,6 @@ public function test_includes() {
 
 		$this->assertFileExists( GIVE_PLUGIN_DIR . 'includes/admin/upgrades/class-give-updates.php' );
 		$this->assertFileExists( GIVE_PLUGIN_DIR . 'includes/admin/upgrades/upgrade-functions.php' );
-
+		$this->assertFileExists( GIVE_PLUGIN_DIR . 'includes/gateways/stripe/class-give-stripe.php' );
```

Comment:
@mehul0810 We check all files which is related to stripe instead of just main file.

Reply:
@ravinderk I've not included other stripe files as this is unit test file for `give.php`, not `class-give-stripe.php`. Hence, I'll add those included files unit test when I will create unit tests for `class-give-stripe.php`